_people/LionelRobert.md

```diff
@@ -0,0 +1,13 @@
+---
+first_name: Lionel P.
+last_name: Robert Jr.
+title: Associate Professor of Information, School of Information and director of the Michigan Autonomous Vehicle Research Intergroup Collaboration
+school: University of Michigan
+website: https://sites.google.com/a/umich.edu/lionelrobert/
+image: https://robotics.umich.edu/wp-content/uploads/2018/12/Lionel_Robert.jpg
+---
+Prof. Lionel Robert is an Associate Professor at the University of Michigan in the School of Information. An Army vet, Prof. Robert is also the director of the [Michigan Autonomous Vehicle Research Intergroup Collaboration](https://mavric.si.umich.edu/). In his own words he focuses on “collaboration through and with technology”, studying trust and teamwork in human-machine teams, including robots and autonomous vehicles. 
```

Comment:
Style we have been using would be Prof. Robert is ..., and then in the bio text here we don't need to repeat the "Associate Professor at the University of Michigan in the School of Information" part, just go right into the army vet and director info.